tests/canary_upgrade_test.go

```diff
@@ -187,14 +195,15 @@ var _ = Describe("[Serial][sig-operator]virt-handler canary upgrade", func() {
 			},
 		})
 
-		err := updateVirtHandler()
+		err = updateVirtHandler()
 		Expect(err).ToNot(HaveOccurred())
 
+		updateTimeout := time.Duration(canaryTestNodeTimeout * nodesToUpdate)
```

Comment:
Do you need to cast it `Duration`?

Reply:
vim-go complains a lot if I don't, so it does seem necessary.